core/src/core/draw.cpp

```diff
@@ -494,6 +514,10 @@ svg::object connected_info_box(
              text_, text_font_);
     ib.add_object(te);
 
+
+    auto img = image_box(id_ + "pull_distr", "../../../css/img/" + id_ + ".png");
+    ib.add_object(img);
```

Comment:
Also here, I would like to have this configurable - the string should not indicate that this is a pull distribution (because you want to